lib/ansible/modules/files/template.py

```diff
@@ -53,6 +53,18 @@
     required: false
     choices: [ "yes", "no" ]
     default: "no"
+  decode:
+    description:
+      - read template using a different encoding (iso8859-15, cp1252)
+    required: false
+    default: "utf-8"
+    version_added: "2.3"
+  encode:
+    description:
+      - write file using a alternative encoding (iso8859-15, cp1252)
```

Comment:
All descriptions start with uppercase and end with a point. Please follow the used standards for consistency.